pkg/process/procutil/data_scrubber_windows.go

```diff
@@ -20,3 +25,61 @@ var (
 	// it's for handling parameters like `/p` and `/rp` in windows
 	forbiddenSymbolsRegex = "[^/a-zA-Z0-9_*]"
 )
+
+var winDotExec = []string{".com", ".exe", ".bat", ".cmd", ".vbs", ".vbe", ".js", ".jse", ".wsf", ".wsh", ".psc1", ".ps1"}
+
+// stripArguments identifies windows extension and extracts command. Otherwise, returns the first element of the cmdline before first space.
+func (ds *DataScrubber) stripArguments(cmdline []string) []string {
+	strCmdline := cmdline[0]
+
+	// Case 1: OS has already completed splitting as there is one token per element.
+	if len(cmdline) > 1 {
+		return cmdline
+	}
+
+	// Case 2: One string for cmdline, use extensionParser() to find first token.
+	if len(cmdline) == 1 && !strings.HasPrefix(strCmdline, "\"") {
+		strippedCmdline, foundExtension := extensionParser(strCmdline, winDotExec)
+
+		if foundExtension {
+			return []string{strings.TrimSuffix(strippedCmdline, " ")}
+		}
+
+		// If no extension is found, return first token of cmdline.
+		return []string{strings.TrimSuffix(strings.SplitN(strippedCmdline, " ", 2)[0], " ")}
```

Comment:
Not sure if this line is needed (this line is here as the original PR on branch calde/proc-1451-strip-arg accounted for the case where the cmdline array only has one single string and does not begin with a drive volume C:\).

EX: cmdline = "python ~/test/run.py -open_password=admin"